public/script/workTrolly.js

```diff
@@ -0,0 +1,15 @@
+$("#createNewTask").ready(function() {
+    $("#createNewTask").hide(0);
+    var date = new Date();
+    document.getElementById("creationDateCreateNewTask").innerHTML = date.getDate() + "/" + (date.getMonth()+1) + "/" + date.getFullYear();
+});
+
+$("#createNewTaskButton").click(function () {
+    $("#createNewTask").show();
+       
+});
+
+$("#backToMainFromCreateNewTask").click(function() {
+    $('#createNewTask').hide(100);
+    $("#bodyContent").fadeIn(100);
+});
```

Comment:
I guess a method can be created named uiloader that can handle all the ui that is shown at current page.

We can discuss on this topic

Reply:
okay, I will do research about it and try to implement it.